src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/DeclarationTests.cs

```diff
@@ -228,8 +231,12 @@ .maxstack  4
   IL_0037:  call       ""string Microsoft.VisualStudio.Debugger.Clr.IntrinsicMethods.GetVariableAddress<string>(string)""
   IL_003c:  ldloc.2
   IL_003d:  stind.ref
-  IL_003e:  ret
-}");
+  IL_003e:  ldloc.1
+  IL_003f:  ldloc.2
+  IL_0040:  newobj     ""System.ValueTuple<int, string>..ctor(int, string)""
```

Comment:
I didn't understand what caused this change. The emitter is only supposed to emit the construction of the ValueTuple result if it is `used`.